src/services/EdFi.DataManagementService.Api.Tests.Unit/Modules/MetaDataModuleTests.cs

```diff
@@ -74,11 +124,11 @@ public async Task MetaData_Returns_Resources_Content()
         using var client = factory.CreateClient();
 
         // Act
-        var response = await client.GetAsync("/metadata/resources/swagger.json");
+        var response = await client.GetAsync("/metadata/specifications/resources-spec.json");
         var content = await response.Content.ReadAsStringAsync();
 
         var jsonContent = JsonNode.Parse(content);
-        var sectionInfo = jsonContent?["Value"]?["info"]?.GetValue<string>();
```

Comment:
You'll notice that I removed `Lazy` on the content provider. I do not see any reason for this to be Lazy, and in fact I discovered an existing bug because of it: some endpoints were formerly returning the actual contents nested inside the serialized Lazy - which you can see here: `Value` should not have been part of the JSON.